ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/ConfigCommitter.scala

```diff
@@ -136,6 +152,13 @@ private[kvutils] case class ConfigCommitter(
         .build
     )
 
+    ctx.set(
+      configDedupKey(ctx.getParticipantId, result.submission.getSubmissionId),
+      DamlStateValue.newBuilder
```

Comment:
make this a constant.

Reply:
Not sure I follow, this is a constant, and its application is wrapped in a function

Reply:
Meant that only compute this value once (e.g. `val dedupValue = DamlStateValue.newBuilder`). This doesn't make sense anymore as we have the record time there.